lambda/getProductsList.ts

```diff
@@ -0,0 +1,23 @@
+import { mockProducts } from './mockData';
+
+const headers = {
+  "Content-Type": "text/plain",
+  "Access-Control-Allow-Origin" : "*",
+};
+
+export const handler = async (event: any) => {
+  try {
+    return {
+      statusCode: 200,
+      headers,
```

Comment:
the same. please use `"content-type": "application/json"` for the 200 OK response.